genome_kit/vcf_table.py

```diff
@@ -733,11 +739,14 @@ def build_vcfbin(outfile,
 
         Missing values are also treated differently: GT values must be specified as a diploid.
         If a single missing allele (e.g. ``0/.``, ``./0``, ``1/.``, or ``./1``) should be encoded as a *possible*
-        :py:const:`VCFTable.GT_HETEROZYGOUS` instead of :py:const:`VCFTable.GT_UNKNOWN`, the missing value should
-        be set to :py:const:`VCFTable.GT_HETEROZYGOUS`. ``./.`` will still be interpreted as
+        :py:const:`VCFTable.GT_HETEROZYGOUS_UNPHASED` instead of :py:const:`VCFTable.GT_UNKNOWN`, the missing value should
+        be set to :py:const:`VCFTable.GT_HETEROZYGOUS_UNPHASED`. ``./.`` will still be interpreted as
         :py:const:`VCFTable.GT_UNKNOWN`::
 
-            {"GT" : np.int8(VCFTable.GT_HETEROZYGOUS)}
+            {"GT" : np.int32(VCFTable.GT_HETEROZYGOUS_UNPHASED)}
```

Comment:
If using int8, you get this error

    ValueError: src/py_variant_table.cpp:461: FORMAT field
    src/variant_table.cpp:554: FORMAT ID GT must be one of GT_UNKNOWN or GT_HETEROZYGOUS_UNPHASED.

comes from the dtype check [here](https://github.com/deepgenomics/GenomeKit/blob/8e8fca0237d647561637ef44fd7dc89b4370ee4a/src/variant_table.cpp#L544):
```
GK_CHECK(*dtype == int32 && (val == (int)gt_unknown || val == (int)gt_heterozygous), value,
			 "FORMAT ID GT must be one of GT_UNKNOWN or GT_HETEROZYGOUS.");
```

Reply:
can we change the dtype to int8 (or cover all the ints).

That + the constant type change will make the interface slightly more natural (no casting of the constant)